src/main/jhipster/src/main/webapp/app/germplasm-manager/inventory/lot-creation-dialog.component.html

```diff
@@ -59,7 +59,7 @@
 			<h4 class="modal-title">Deposit</h4>
 			<div class="checkbox">
 				<label>
-					<input type="checkbox" class="form-check-input" [(ngModel)]="initialDeposit" name="depositAvailable" id="depositAvailable">
+					<input type="checkbox" class="form-check-input" [(ngModel)]="initialDeposit" name="depositAvailable" id="depositAvailable" [disabled]="studyId">
```

Comment:
Is it better to hide the "Initial Deposit" checkbox and label instead of disabling it when studyId is present?

Reply:
I think it makes sense. Done.